### ISSUE TITLE
Evaluator - select for grading - egy jkv lefoglalasa

### ISSUE THREAD

**Issue by lordblendi** [bme-db-lab/szglab5-frontend#26]
Rakattintas a jkv-re, elohozza az adott beadasrol az osszes informaciot
- hallgato informacio: nev, neptun, feladattipus, labortipus, beadas ideje, hatarido, laborvezeto + email, laborcsoport
- jkv letoltesenek lehetosege

Egy **button** arra, hogy vegleges legyen a lefoglalas.

**Comment by csutorasr** [comment]
[Ebben a commitban](https://github.com/bme-db-lab/szglab5-frontend/commit/3fbcc8dd21d9bca7dcb0bc046013403dc4593e18) modostitottam a foglalas gombra.

A jegyzokonyv letoltes kell oda?

Meg hianyzo adatok: beadas ideje, hatarido, laborvezeto + email, laborcsoport.

Beadas idejet, hataridot kivalasztas elott is latta. (ha olyan az ido ra se megy)

labcsoport kb lehetetlen dolog. Szerintem semmilyen vegpont nincs meg a kozeleben sem. :(

**Comment by lordblendi** [comment]
Ok, ha a labor csoport nem megy, azt hagyd.

Letoltes gomb hasznos, azt rakj oda kerlek.

Beadas ideje es hatarido kerlek jelenjen meg itt is (hatha felre kattint) a
user.

**Comment by csutorasr** [comment]
Lefoglalas elott letolteni? Biztos? :)

**Comment by lordblendi** [comment]
Igen. Belenezhetsz, hogy milyen a munka mielott lefoglalod. :) ua, mint
lementeni, megnezni es visszarakni

**Comment by csutorasr** [comment]
A hataridot nem kell megjeleniteni, mert ha olyanra megy ami meg nem erte el, akkor arra figyelmeztet es nem engedi lefoglalni. (`This event has not reached its deadline.`)

A letoltes gombot most rakom hozza.

**Comment by lordblendi** [comment]
1. Olyanra ne mehessen, ami meg nem erte el a hataridot, mert annak meg se kellene jelennie a tablazatban, ahol foglalhatsz. 
2. A hataridot meg kell jeleniteni, hogy tudja a javito, mennyit kesett pl a diak a beadassal  

Ahogy irtam fentebb, kerlek tedd oda a hataridot.

**Comment by csutorasr** [comment]
A haratidot [kiirja](https://github.com/bme-db-lab/szglab5-frontend/commit/92e583ab34e54ce045bc3ed6462adf31d687d3ca#diff-88c641fd389e2aa22f0274edc6f202a5R30).

**Comment by csutorasr** [comment]
Waiting for backend.

**Comment by csutorasr** [comment]
@Kisfejes Zárd le ha készen van.
